pwndbg/commands/canary.py

```diff
@@ -14,8 +15,25 @@
 
 DEFAULT_NUM_CANARIES_TO_DISPLAY = 1
 
+# Architecture-specific TLS canary offsets
+# These offsets are from the TLS base to the canary
+# References:
+# - x86_64: fs:0x28 (https://elixir.bootlin.com/glibc/latest/source/sysdeps/x86_64/nptl/tls.h)
```

Comment:
Can we change the links to not be 'latest' but to latest current version? This way we know when it worked last time if it ever changes